linkerd/app/outbound/src/test_util.rs

```diff
@@ -1,31 +1,21 @@
-use crate::{
-    endpoint::{TcpConcrete, TcpLogical},
-    Config,
-};
-use futures::prelude::*;
-use ipnet::IpNet;
+use crate::Config;
+pub use futures::prelude::*;
+pub use ipnet::IpNet;
 use linkerd2_app_core::{
-    config, exp_backoff, profiles,
-    proxy::http::h2,
-    svc::NewService,
-    transport::{listen, tls},
-    Error, IpMatch,
+    config, exp_backoff, profiles, proxy::http::h2, transport::listen, IpMatch,
 };
-use linkerd2_app_test as test_support;
+pub use linkerd2_app_test as support;
```

Comment:
nit/TIOLI: might be slightly nicer to just
```suggestion
pub use linkerd2_app_test::*;
```
 and then have the tests just use _this_ module as `support`? not a blocker.

Reply:
maybe in a followup. glob imports end up being pretty hairy to understand, and in this case it's easy for unintentional overlaps like `io() vs transport::io`